docker-compose.yaml

```diff
@@ -1,14 +1,14 @@
 services:
   grafana:
     user: root
-    container_name: 'axiom-axiomgrafana-datasource'
+    container_name: 'axiomhq-axiom-datasource'
```

Comment:
Is this generated? The name looks funny

Reply:
this is the name used in [Grafana marketplace](https://grafana.com/grafana/plugins/axiomhq-axiom-datasource/). Don't know how did that start but it seems to follow this patter: `$orgName-$pluginName-datasource` but I checked other plugins like splunk and honeycomb, they are named: `grafana-splunk-plugin` and `grafana-honeycomb-plugin`